haskell-indexer-backend-ghc/src/Language/Haskell/Indexer/Backend/Ghc.hs

```diff
@@ -543,6 +544,24 @@ relationsFromRenamed ctx declAlts (hsGroup, _, _, _) =
         let fromAlt = altTickToPrimary declAlts
         in Relation (fromAlt s) k (fromAlt t)
 
+-- | Exports module imports.
+importsFromRenamed :: GhcMonad m => ExtractCtx -> RenamedSource -> m [ModuleTick]
+importsFromRenamed ctx (_, lImportDecls, _, _) = mapM mkImport lImportDecls
+  where
+    mkImport :: GhcMonad m => LImportDecl Name -> m ModuleTick
+    mkImport (L _ implDecl) = do
+      pkgModule <- extractPkgModule . unLoc . ideclName $ implDecl
+      let pkgSpan = give ctx (srcSpanToSpan . getLoc $ ideclName implDecl)
+      return $ ModuleTick pkgModule pkgSpan
+
+    extractPkgModule :: GhcMonad m => ModuleName -> m PkgModule
+    extractPkgModule name = do
+      m <- findModule name Nothing
```

Comment:
Please reuse [`extractModuleName`](https://github.com/google/haskell-indexer/blob/a9eb13947774419e29bc1128d91d13a1558665a6/haskell-indexer-backend-ghc/src/Language/Haskell/Indexer/Backend/Ghc.hs#L1191) somehow, to use the shared parts and have the package demangling applied.